app/screens/settings-screen/username-screen.tsx

```diff
@@ -202,18 +187,14 @@ export const UsernameScreen = ({ navigation }: Props) => {
         leftIcon={{ type: "ionicon", name: "ios-person-circle" }}
         onChangeText={onChangeText}
         errorStyle={styles(messageIsError).error}
-        errorMessage={message}
+        errorMessage={loadingUserNameExist ? "" : message}
```

Comment:
I'm surprised not to see the `shouldShowErrorMessage` variable used here. Perhaps `loadingUserNameExists` needs to be taken into account in the logic that determines the value of that variable.

Reply:
It's a little clunky to update `shouldShowErrorMessage` with `loadingUserNameExists`. I think it would be cleanest to rename `shouldShowErrorMessage` to `shouldShowCharacterMinimumErrorMessage`.